_pages/en_US/installing-boot9strap-(soundhax-sp).txt

```diff
@@ -0,0 +1,33 @@
+---
+title: "Installing boot9strap (Soundhax)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+For technical details on the exploits that you will be using on this page, see [here](https://github.com/nedwill/soundhax) (Soundhax) and [here](https://github.com/TuxSH/universal-otherapp) (universal-otherapp).
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+### Compatibility Notes
+
+This Soundhax method (when combined with universal-otherapp) is compatible with versions 1.0.0-0 through 11.10.0-36 in all regions. If the two numbers before the region in the system version string is equal to or more than 37, Soundhax is patched and won't work.
+
+{% include_relative include/soundhax.txt sp= "true" %}
+
+#### Section III - Installing boot9strap
```

Comment:
why not just put the whole thing in the include? You can do include_relative from the include too, so we could just cut this entire page and throw it in the `soundhax.txt` file, this way if we ever add a section to this page (highly unlikely) we don't have to edit two pages

Reply:
done